rep-2010.rst

```diff
@@ -234,7 +234,10 @@ Any existing tooling for aiding users in setting up ROS security should consider
 Command line interface
 ----------------------
 
-TODO: Consider proposing an interface for a CLI here. Although it seems a bit tangent to the core feature, I think it is important to the proposal since it makes it so people can immediately start using the feature.
+We can have a tool like ``ros2 service echo example_interfaces/srv/AddTwoInts`` that internally subscribes to the two hidden topics ``example_interfaces/srv/AddTwoInts_RequestEvent`` and
+``example_interfaces/srv/AddTwoInts_ResponseEvent`` and echoes them. It should also support additional command line parameters used for topics, like ``--filter, --no-arr, --no-str, --include-msg-info``, etc.
```

Comment:
Also, once we echo the service how will we know that whether it is `AddTwoInts_ResponseEvent` or `AddTwoInts_RequestEvent` I suppose their will be a heading after echoing on the command line, just curious.

Reply:
Added an example to clarify. The responses and requests can be separated by the ``request_type`` fields.